packages/serverpod/lib/src/generated/readwrite_test.dart

```diff
@@ -39,12 +39,10 @@ abstract class ReadWriteTestEntry extends _i1.TableRow {
 
   @override
   _i1.Table get table => t;
-
   ReadWriteTestEntry copyWith({
     int? id,
     int? number,
   });
-
   @override
```

Comment:
I don't understand why it sometimes removes whitespaces between methods....

Reply:
I don't get this either... it seems to be happening on methods.